pkg/multus/multus.go

```diff
@@ -486,6 +488,59 @@ func cmdErr(k8sArgs *types.K8sArgs, format string, args ...interface{}) error {
 	return logging.Errorf(prefix+format, args...)
 }
 
+func convertResourceClaimToAnnotations(kubeClient *k8s.ClientInfo, pod *v1.Pod) error {
+	for _, rc := range pod.Spec.ResourceClaims {
```

Comment:
add warn logs that it ignores the existed annotation owing to DRA claim, if multus annotation exists

Reply:
done